### DISCUSSION TITLE
bug: Deleting a keymap set by LazyVim.toggle.map doesn't remove it from which-key

### DISCUSSION THREAD

**Discussion by nowaylifer** [LazyVim/LazyVim#4182] [Q&A]
### Did you check docs and existing issues?

- [X] I have read all the LazyVim docs
- [X] I have updated the plugin to the latest version before submitting this issue
- [X] I have searched the existing issues of LazyVim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

0.10.1

### Operating system/version

Arch Linux

### Describe the bug

I deleted default keymap for toggling window maximazing ("<leader>wm")  which is set by LazyVim.toggle.map, but it didn't result in removing corresponding which-key entry as well.

### Steps To Reproduce

1. Delete keymap for toggling window maximazing:  `vim.keymap.del("n", "<leader>wm")`.
2. Observe existense of "<leader>wm" entry in which-key.

### Expected Behavior

Corresponding which-key entry is removed.

**Comment by EricDriussi** [comment]
Same issue here.
Unfortunately given [this response](https://github.com/LazyVim/LazyVim/discussions/4062#discussioncomment-10055555) I'm inclined to think this is intended behavior.
It would be cool to have this work as you describe tho.
